app/services/video-recorder.js

```diff
@@ -347,7 +347,7 @@ const VideoRecorder = Ember.Object.extend({
                         resolve(); // skip mic check and resolve the install promise
                     }
                 } else {
-                    reject();
+                    reject(new Error(`Mic check error: no input stream.`));
                 }
             };
```

Comment:
Here we're throwing an error in the reject call.  Just an inconsistency I noticed.

Reply:
You've explained this, thank you!